packages/vite/src/node/plugins/importAnalysis.ts

```diff
@@ -650,16 +651,22 @@ export function importAnalysisPlugin(config: ResolvedConfig): Plugin {
             }
 
             if (!ssr) {
-              const url = rawUrl.replace(cleanUpRawUrlRE, '').trim()
+              // Remove comments and trim whitespace
+              const cleanedUrl = stripLiteral(rawUrl)
+              const match = cleanedUpRawUrlRE.exec(cleanedUrl)
```

Comment:
Could we rename `cleanedUpRawUrlRE` to something like `trimWhitespaceRE`? I think it's a bit unclear what it does until I read the regex.

Reply:
I extracted the util so it is more clear in the caller side here https://github.com/vitejs/vite/pull/14910/commits/e814f7f7127b194edf4f4a19845cafc5192df0f5